packages/nodes-base/nodes/HttpRequest/GenericFunctions.ts

```diff
@@ -88,7 +88,14 @@ export function sanitizeUiMessage(
 			),
 		};
 	}
-
+	//redact headers with Base64 encoded credentials
```

Comment:
Nit: I would not comment about base64 here, the code is redacting the authorization header, not specifically base64-ed credentials